pertpy/_utils.py

```diff
@@ -0,0 +1,54 @@
+from pathlib import Path
+from textwrap import dedent
+
+import matplotlib.pyplot as plt
+from matplotlib.pyplot import Figure
+
+
+def savefig_or_show(
```

Comment:
Personally, I'm not a big fan of including an option to save figures in the function signature... If you have `return_fig`, it's as easy as

```python
fig = plot_something(return_fig=True)
fig.savefig("file.pdf")
```

to save a figure and specify dpi, format etc. at the same time.

Reply:
Fair, that's also a relict of the scanpy API I wager. Let's go for `return_fig` only